app.js

```diff
@@ -13,4 +16,268 @@ navLinks.forEach((link) => {
+});
+
+// serving main conetent dynamically
+
+const cardContent = [
+  {
+    id: 'cardOne',
+    title: 'Tonic',
+    fields: ['Canopy', 'Back End Dev', '2015'],
+    description: 'A daily selection of privately personalized reads. no accounts or sign-ups are required.',
+    stack: ['html', 'css', 'javascript'],
+    images: {
+      contentImage: 'images/Snapshoot-Portfolio.png',
+      counter: 'images/Counter.png',
+    },
+    button: 'See Project',
+  },
+
+  {
+    id: 'cardTwo',
+    title: 'Multi-Post Stories',
+    fields: ['Canopy', 'Back End Dev', '2015'],
+    description: 'A daily selection of privately personalized reads. no accounts or sign-ups are required.',
+    stack: ['html', 'css', 'javascript'],
+    images: {
+      contentImage: 'images/Snapshoot-Portfolio1.png',
+      counter: 'images/Counter.png',
+    },
+    button: 'See Project',
+  },
+
+  {
+    id: 'cardThree',
+    title: 'Multi-Post Stories',
+    fields: ['Canopy', 'Back End Dev', '2015'],
+    description: 'A daily selection of privately personalized reads. no accounts or sign-ups are required.',
+    stack: ['html', 'css', 'javascript'],
+    images: {
+      contentImage: 'images/Snapshoot-Portfolio2.png',
+      counter: 'images/Counter.png',
+    },
+    button: 'See Project',
+  },
+
+  {
+    id: 'cardFour',
+    title: 'Multi-Post Stories',
+    fields: ['Canopy', 'Back End Dev', '2015'],
+    description: 'A daily selection of privately personalized reads. no accounts or sign-ups are required.',
+    stack: ['html', 'css', 'javascript'],
+    images: {
+      contentImage: 'images/Snapshoot-Portfolio3.png',
+      counter: 'images/Counter.png',
+    },
+    button: 'See Project',
+  },
+];
+
+cardContent.forEach((item) => {
+  mainContainer.innerHTML += `<div class="content">
+    <img class="item item1" src="${item.images.contentImage}" alt="first project image">
+    <div class="card">
+        <h2 class="item item2">${item.title}</h2>
+        <div>
+            <ul class="row">
+                <li class="item item3">${item.fields[0]}</li>
+                <li><img class="item item4" src=" ${item.images.counter}" alt="a dot image"></li>
+                <li class="item item5">${item.fields[1]}</li>
+                <li><img class="item item6" src="${item.images.counter}" alt="second dot image"></li>
+                <li class="item item7">${item.fields[2]}</li>
+            </ul>
+        </div>
+        <p class="item item8">${item.description}</p>
+        <div class="list">
+            <ul class="skills">
+                <li class="html stack"><button>${item.stack[0]}</button></li>
+                <li class="css stack"><button>${item.stack[1]}</button></li>
+                <li class="javascript stack"><button>${item.stack[2]}</button></li>
+            </ul>
+        </div>
+        <div class="click">
+            <button btn-id="${item.id}" class="btn showBtn" type="button">${item.button}</button>
+        </div>
+    </div>
```

Comment:
- As requested in the project requirements:

<img src="https://user-images.githubusercontent.com/42799579/209001206-5d022bd2-75fe-4891-8dc1-e0915b5d9106.png">
  
- [ ] Kindly remember that you need to follow the original design 😉. Currently, all of your work cards are being displayed in the same way (image on the left & text on the right). Please notice that, in the original design, the project cards alternate between an image on the left & image on the right:

<table>
<th>Your design 👇</th>
</table>

![sul-mob](https://user-images.githubusercontent.com/78634841/209326663-3c05046c-b6c1-46ce-a161-bb02532fa08b.png)

<table>
<th>Figma design 👇</th>
</table>

![card](https://user-images.githubusercontent.com/78634841/209327025-d99e43e3-fdb4-410e-b17f-9cfb3c310181.png)